gensim/models/doc2vec.py

```diff
@@ -649,6 +649,7 @@ def clear_sims(self):
         """Resets the current word vectors. """
         self.wv.vectors_norm = None
         self.wv.vectors_docs_norm = None
+        self.docvecs.vectors_docs_norm = None
```

Comment:
can you please check and verify that "similar" thing happens in w2v\fasttext models too?

Reply:
Yes. If you look in `word2vec.py` you see that there are actually two functions `_clear_post_train` and `clear_sims` which do the same thing (one of the functions is therefore redundant): Setting `self.wv.vectors_norm = None`.
The same thing (setting `self.wv.vectors_norm = None` and more) also happens in `fasttext.py` in function `_clear_post_train`.
Does this solve your concerns or do you want a more detailed explanation?

Reply:
@menshikh-iv I am not sure, whether I should resolve the conversation or wait for a response by you. What is preferred here (or in general in such cases)?

Reply:
Separately, the line above this (which existed prior to your edits) – `self.wv.vectors_docs_norm = None` – looks fishy. A normal `wv` set of word-vectors wouldn't be expected to even have a `vectors_docs_norm` property, unless some other code has mistakenly poked doc-vectors into there.